Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py

```diff
@@ -0,0 +1,238 @@
+        """Report email reputation using the '/report' API endpoint"""
+        request_params: Dict[str, Any] = {}
+        request_params["email"] = email
+        request_params["tags"] = tags
+
+        if description:
+            request_params["description"] = description
+
+        if timestamp:
+            request_params["timestamp"] = timestamp
+
+        if expires:
+            request_params["expires"] = expires
+
+        return self._http_request(
+            method='POST',
+            url_suffix='/report',
+            json_data=request_params
+        )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Test API connectivity and authentication."""
+    try:
+        client.get_email_address_reputation(email="test@example.com")
+    except DemistoException as e:
+        if 'invalid api key' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return 'ok'
+
+
+def email_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Get email address reputation from EmailRepIO and calculate DBotScore.
+
+    DBot score:
+    Good: Suspicious = false
+    Malicious: Suspicious = true + (malicious_activity_recent = true or credentials_leaked_recent = true)
+    Suspicious: Suspicious = true and not malicious
+    """
+    emails = argToList(args.get('email'))
+    if len(emails) == 0 or emails is None:
+        raise ValueError('Email(s) not specified')
+
+    email = emails[0]
+    email_data = client.get_email_address_reputation(email)
+    description = f'{INTEGRATION_NAME} returned'
+    suspicious = email_data.get('suspicious')
+    malicious_activity_recent = email_data.get('details.malicious_activity_recent')
+    credentials_leaked_recent = email_data.get('details.credentials_leaked_recent')
+    if not suspicious:
+        score = Common.DBotScore.GOOD
+        description = ''
+    elif malicious_activity_recent or credentials_leaked_recent:
+        if malicious_activity_recent:
+            description += ' malicious_activity_recent'
+        if credentials_leaked_recent:
+            description += ' credentials_leaked_recent'
+        score = Common.DBotScore.BAD
+    else:
+        score = Common.DBotScore.SUSPICIOUS
+        description = ''
+
+    dbot_score = Common.DBotScore(
+        indicator=email,
+        indicator_type=DBotScoreType.ACCOUNT,
+        integration_name=INTEGRATION_NAME,
+        score=score,
+        malicious_description=description
+    )
+
+    account_context = Common.Account(
+        id=email,
+        email_address=email,
+        dbot_score=dbot_score
+    )
+
+    readable_output = tableToMarkdown('Email', email_data)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_NAME}.Email',
+        outputs_key_field='id',
+        outputs=email_data,
+        indicators=[account_context]
```

Comment:
```suggestion
        indicator=account_context
```

Reply:
Is this change already merged? My tests seem to fail

Reply:
yeah i think so